package.json

```diff
@@ -92,6 +92,27 @@
         "onView:azureWorkspace"
     ],
     "contributes": {
+        "x-azResourcesV2": {
+            "application": {
+                "branches": [
+                    {
+                        "type": "AzureCosmosDb"
+                    }
+                ]
+            },
+            "workspace": {
+                "branches": [
+                    {
+                        "type": "ms-azuretools.vscode-cosmosdb"
```

Comment:
Maybe something like `AttachedDatabaseAccount` is better here. @bwateratmsft thoughts?

Reply:
That sounds reasonable to me